From a5b9f8a1259f43d49d8184886765e79a56af8495 Mon Sep 17 00:00:00 2001
From: Abhishek Koneru <akoneru@redhat.com>
Date: Thu, 13 Nov 2014 08:02:58 -0500
Subject: [PATCH] Removed pyenv references in barbican.sh

Removed pyenv related variables in barbican.sh for packaging
in fedora.

Change-Id: I1048867b287aa2502671ef8f8a3de5e19186ee8b
---
 bin/barbican.sh | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/bin/barbican.sh b/bin/barbican.sh
index c661890ee4439b794771130fc67d1c7822e9fff1..3e1f4603fef76759b0eb5a0d159e8bb7ede78b42 100755
--- a/bin/barbican.sh
+++ b/bin/barbican.sh
@@ -3,10 +3,6 @@
 CONFIG_DIR=/etc/barbican
 DB_DIR=/var/lib/barbican
 
-# VIRTUAL_ENV is set by virtualenv on activate. If VIRTUAL_ENV is not,
-# available, it attempts to fallback on pyenv for the python environment path.
-VENV_DIR=${VIRTUAL_ENV:-`pyenv prefix`}
-
 LOCAL_CONFIG_DIR=./etc/barbican
 if [ ! -d $LOCAL_CONFIG_DIR ];
 then
@@ -46,7 +42,7 @@ debug_barbican()
     echo "Starting barbican in debug mode ..." --pydev-debug-host $debug_host --pydev-debug-port $debug_port
     PYDEV_DEBUG_PARAM="--env PYDEV_DEBUG_HOST=$debug_host --env PYDEV_DEBUG_PORT=$debug_port"
 
-    uwsgi --master --emperor $CONFIG_DIR/vassals -H $VENV_DIR $PYDEV_DEBUG_PARAM
+    uwsgi --master --emperor $CONFIG_DIR/vassals
 }
 
 start_barbican()
@@ -54,7 +50,7 @@ start_barbican()
     # Start barbican server up.
     #   Note: Add ' --stats :9314' to run a stats server on port 9314
     echo "Starting barbican..."
-    uwsgi --master --emperor $CONFIG_DIR/vassals -H $VENV_DIR
+    uwsgi --master --emperor $CONFIG_DIR/vassals
 }
 
 stop_barbican()
-- 
1.8.5.3

